index.js

```diff
@@ -65,13 +64,19 @@ app.use('/', auth, cors(), morgan('combined'), graphqlHTTP(request => {
 
   const accessToken = request.headers['x-access-token'];
   const userID = request.headers['x-user-id'];
+  const tz = request.headers['x-timezone'];
+  let defaultTimezone;
+  if (tz && tz.split(';')[2].length) {
```

Comment:
I think this second condition should come after something like `tz.split(';').length >= 3`, otherwise you run the risk of a malformed string to cause an exception when calling `length` on it.